src/seedu/addressbook/parser/Parser.java

```diff
@@ -259,7 +259,34 @@ private Command prepareFind(String args) {
      */
     private Command prepareSwap(String args) {
         //TODO
```

Comment:
Remove the TODO since you have already done it. Failure to remove will result in team mates seeing this task as a TODO on intelliJ

Reply:
Noted! Thanks for the feedback.